lib/digitizer_block_impl.cc

```diff
@@ -912,7 +912,7 @@ namespace gr {
    digitizer_block_impl::poll_work_function()
    {
      boost::unique_lock<boost::mutex> lock(d_poller_mutex, boost::defer_lock);
-     auto poll_rate = boost::chrono::microseconds((long)(d_poll_rate * 1000000));
+     auto poll_rate = std::chrono::microseconds((long)(d_poll_rate * 1000000));
```

Comment:
`poll_rate` naming is a bit confusing since it's not a 'rate' (e.g. in units of [1/s]) but a `duration' (i.e. in units of [ms]).

The `std::chrono::microseconds((long)(d_poll_rate * 1000000))` should be replaced by a `std::chrono::duration_cast` to make the from/to units more explicit.

Reply:
`std::chrono::duration_cast<std::chrono::microseconds>(std::chrono::seconds(d_poll_rate));` wont work, because std::chrono::seconds only accepts integers.

I just removed the auto attribute and used double as internal type, so that seconds can be stored .. fine like that ?

Validated like this:
```
     auto poll_duration = std::chrono::microseconds((long)(d_poll_rate * 1000000));
     std::chrono::duration<double> poll_duration2(d_poll_rate);
     std::cout << "poll_duration in us " << std::chrono::duration_cast<std::chrono::microseconds>(poll_duration).count() << std::endl;
     std::cout << "poll_duration2 in us " << std::chrono::duration_cast<std::chrono::microseconds>(poll_duration2).count() << std::endl;
```
Haha, will printing thecount and only getting a zero, I just found out why that whole poll_rate wont work :D https://github.com/fair-acc/gr-flowgraph/pull/6

Reply:
Actually the name `poll_rate` is already missleading in the flowgraph. Though on the block the added unit `(s)` shows what should be filled in.